ui-react/packages/atlasmap/src/UI/useDoubleSelectDialog.tsx

```diff
@@ -0,0 +1,179 @@
+import {
+  Modal,
+  Button,
+  FormSelect,
+  FormSelectOption,
+  Label,
+} from "@patternfly/react-core";
+import React, { ReactPortal, useCallback, useRef, useState } from "react";
+import { createPortal } from "react-dom";
+import { css, StyleSheet } from "@patternfly/react-styles";
+import { CancelCallback } from "./useSingleInputDialog";
+
+interface ISelectValues1 {
```

Comment:
Maybe export this and import it where you use it instead of redeclaring it?

Reply:
will do when the class name select is implemented.

Reply:
corrected - file removed.